src/css/register.css

```diff
@@ -0,0 +1,102 @@
+@import url("https://fonts.googleapis.com/css?family=Dosis:400,600,700");
+/*------- MIXINS--------*/
+/*------- EXTEND CLASSES--------*/
+@import url("https://fonts.googleapis.com/css?family=Dosis:400,600,700");
+.text-center {
+  text-align: center; }
+
+.a-decorate {
+  text-decoration: none !important; }
+
+.cursor-link {
+  cursor: pointer; }
+
+.float-left {
+  float: left; }
+
+.float-right {
+  float: right; }
+
+.basic-header {
+  font-weight: 700;
+  letter-spacing: -1px; }
+
+.basic-desc {
```

Comment:
are these files generated ?

Reply:
These all are generated files. I have removed them and created a single css file main.css.